cnf-certification-test/accesscontrol/suite.go

```diff
@@ -737,19 +737,32 @@ func testNamespaceResourceQuota(env *provider.TestEnvironment) {
 }
 
 const (
-	sshServicePortNumber   = 22
 	sshServicePortProtocol = "TCP"
 )
 
 func testNoSSHDaemonsAllowed(env *provider.TestEnvironment) {
 	var compliantObjects []*testhelper.ReportObject
 	var nonCompliantObjects []*testhelper.ReportObject
 
-	sshPortInfo := netutil.PortInfo{PortNumber: sshServicePortNumber, Protocol: sshServicePortProtocol}
+	sshServicePortNumber := 22
 
 	for _, put := range env.Pods {
 		cut := put.Containers[0]
 
+		// 1. Find SSH port
+		port, err := netutil.GetSSHDaemonPort(cut)
+		if err != nil {
+			logrus.Errorf("error occurred while finding ssh port on %s, err %v", cut, err)
```

Comment:
Need to break here, right?

Reply:
No, continue with port default 22 to check if it is listening to be put into non-compliant list in the end.